packages/zambdas/src/shared/chart-data/index.ts

```diff
@@ -337,7 +337,7 @@ export function makeObservationResource(
   }
 
   if (isObservationTextFieldDTO(data)) {
-    if ('note' in data) {
+    if ('note' in data && data.note) {
```

Comment:
What if somebody tries to delete a note (probably an empty string will be used in this case), will it be possible with this change?